letsencrypt/cli.py

```diff
@@ -38,6 +37,7 @@
 
 from letsencrypt.display import util as display_util
 from letsencrypt.display import ops as display_ops
+from letsencrypt.errors import Error, ConfiguratorError, CertStorageError
```

Comment:
1. Most importantly, consistency with the rest of the project!
2. https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Imports#Imports